diffkemp/diffkemp.py

```diff
@@ -58,6 +59,8 @@ def build_c_project(args):
         if args.clang_drop is not None
         else "",
         wrapper_env_vars["llvm_link"]: args.llvm_link,
+        wrapper_env_vars["no_opt_override"]: ("1" if args.no_opt_override
+                                              else "0")
```

Comment:
Isn't it possible to pass a boolean to the env variable? So that we could do
```suggestion
        wrapper_env_vars["no_opt_override"]: not args.no_opt_override
```

Reply:
It is possible to pass a boolean, but I would still need to do something like this in `cc_wrapper` when loading the value:
```python3
if os.environ.get(wrapper_env_vars["no_opt_override"]) == "True":
        no_opt_override = True
    else:
        no_opt_override = False
```

because passing `False` will pass "False" and `bool("False") == True`

Reply:
I see. Ok, let's stick with `1` and `0`.